GovUkDesignSystemComponents/Enums/CellFormat.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GovUkDesignSystem.GovUkDesignSystemComponents.Enums
+{
+    public enum Format
```

Comment:
I wonder if we should make this name more specific? e.g. TableCellFormat

Reply:
Yes, we should. Done, thanks